testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6332,6 +6346,66 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f0.forEachLibrary(list(SqlLibrary.MYSQL, SqlLibrary.SPARK), consumer);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6325">[CALCITE-6325]
+   * Add LOG function (enabled in MYSQL, Spark library)</a>. */
```

Comment:
nit: MySQL*, and can you make that consistent across this PR (commit message, jira, etc.)

Reply:
I'm not sure what it means, sorry

Reply:
Please write it as "MySQL", that's how they style it.

Reply:
Add LOG function as MySql style  or Add LOG function (enabled in MYSQL*) ?

Reply:
Please write it as "MySQL"

Reply:
Sorry, I was sick during this period and was delayed for some time.